lua/plugins.lua

```diff
@@ -44,7 +44,7 @@ return require("packer").startup(function(use)
   -- Telescope
   use {
     "nvim-telescope/telescope.nvim",
-    config = [[require('core.telescope').setup()]],
+    config = require("core.telescope").setup(),
```

Comment:
Why calling directly setup instead of assigning it as callback?

Reply:
Thanks,  I meant to change it so it would match the style of the others but I forgot to surround it in a function.

Reply:
Ok.  I wrapped it.